core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -172,6 +175,15 @@ default boolean isComplete() {
      */
     CompletableFuture<Void> completionFuture();
 
+    /**
+     * Returns a {@link Throwable} that was captured immediately when abortion or cancellation occurred.
+     * Note that this method will return {@code null} when this streams is not complete yet
+     * or complete successfully. An error cause in {@link #completionFuture()} could be obtained only
+     * when this stream is exceptionally complete.
```

Comment:
I'm not sure what this sentence ('An error cause ...') means. Could you elaborate?